TweetBook/.editorconfig

```diff
@@ -0,0 +1,133 @@
+# To learn more about .editorconfig see https://aka.ms/editorconfigdocs
```

Comment:
I will leave here all my remarks about the git related stuff:

- Please try to add to gitignore the _.vs/TweetBook/v17/.suo_  file, and also the _.editorconfig_ one(since it does not contain any project related settings). I would like for them not to be in the next PR or you could take the time to do it now. They would not appear as changed files in the PR if you work correctly in adding them to the gitignore file.
- I've seen that you have some files present here from the bin folder. For now we will hope that they are no longer tracked by git, and we will see if they appear in the next PR. If they do, we must repeat the process with the untracking for them. So please keep in mind to follow them.

Reply:
Ok, got it !